pjp.tm11/src/main/webapp/login/css/util.css

```diff
@@ -0,0 +1,2993 @@
+/*[ FONT SIZE ]
+///////////////////////////////////////////////////////////
+*/
+.fs-1 {font-size: 1px;}
+.fs-2 {font-size: 2px;}
+.fs-3 {font-size: 3px;}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected single space after "{" of a single-line block (block-opening-brace-space-after)](https://app.codacy.com/manual/ohbus/PJP/pullRequest?prid=5662373)